Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -677,7 +676,12 @@ def send_request(self, path: str, method: str = 'GET', body: dict | None = None,
         body = body if body is not None else {}
         params = params if params is not None else {}
 
-        if sc_api:
+        if custom_api:
+            server_url = demisto.params().get('url')
+            api = '/' + custom_api.strip(' /') + '/'
+            path = path.strip(' /')
+            url = f'{get_server_url(server_url)}{api}{path}'
```

Comment:
Why is this done diffrently rom self._sc_server_url? 
For xsoar supported packs it is very important to keep the code practices.

Reply:
This is the whole reason for this PR. The `self._sc_server_url` consist of 
```
sc_server_url = f'{get_server_url(server_url)}{sc_api}'
```
Because `sc_api` (and `cr_api`) are hardcoded, we need to rebuild the url just like the `sc_server_url` on line 2987 but this time with the provided custom api endpoint instead of the hardcoded `sc_api` endpoint. Building the url is exactly the same as for sc_server_url therefore in my opinion this change keeps the code practices.

Reply:
What about handling the version of the API (received as a parameter) as seen in ServiceNow documentation, and as Implemented with the other endpoints in the main function?